scripts/ssl-manager.js

```diff
@@ -1112,20 +1179,26 @@ function SSLManager(config) {
         ) || "";
     };
 
-    me.scheduleAutoUpdate = function scheduleAutoUpdate() {
-        var fileName = "auto-update-ssl-cert.sh",
-            scriptUrl = me.getScriptUrl(fileName);
+    me.scheduleAutoUpdate = function scheduleAutoUpdate(crontime) {
+        var scriptUrl = me.getScriptUrl(AUTO_UPDATE_SCRIPT);
 
         return nodeManager.cmd([
             "wget --no-check-certificate '%(url)' -O %(scriptPath)",
             "chmod +x %(scriptPath)",
+            "crontab -l | grep -v '/root/.acme.sh' | crontab -",
```

Comment:
why?

Reply:
acme client add cron task by default. There are already acme updates available where such cron presents. This cmd should be presented approximately 2-3 months. Then it can be removed from LE script. For future updates in install-sl.sh script option _--no-cron_ has been added in this pull request